src/pages/EmployeeCreation/EmployeeCreationPage.jsx

```diff
@@ -0,0 +1,122 @@
+import { Avatar, Box, Button, Fab, TextField, Typography } from "@mui/material";
+import UploadBox from "./components/UploadBox";
+import PageLayout from "../../components/PageLayout";
+import { useState } from "react";
+
+/**
+ * The `EmployeeCreationPage` function is a React component that renders a form for users to create
+ * a new employee, including fields for the employee's name, email, phone number, address, and
+ * additional information, as well as a button to upload a resume and a button to create the employee.
+ * @returns {JSX.Element} The Employee creation page component.
+ */
+
+function EmployeeCreationPage() {
+
+  const [employeeName, setEmployeeName] = useState("");
+  const [employeeEmail, setEmployeeEmail] = useState("");
+  const [employeePhoneNumber, setEmployeePhoneNumber] = useState("");
+  const [employeeAddress, setEmployeeAddress] = useState("");
+  const [employeeExtraInfo, setEmployeeExtraInfo] = useState("");
+
+  const handleEmployeeNameChange = (event) => {
+    setEmployeeName(event.target.value);
+  };
+  const handleEmployeeEmailChange = (event) => {
+    setEmployeeEmail(event.target.value);
+  };
+  const handleEmployeePhoneNumberChange = (event) => {
+    setEmployeePhoneNumber(event.target.value);
+  }
+  const handleEmployeeAddressChange = (event) => {
+    setEmployeeAddress(event.target.value);
+  }
+  const handleEmployeeExtraInfoChange = (event) => {
+    setEmployeeExtraInfo(event.target.value);
+  }
+
+  function EmployeeCreation(event) {
```

Comment:
```suggestion
  function handleSubmit(event) {
```

Function names should be camelCase